java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -3318,6 +3318,19 @@ public final ColumnVector getMapKeyExistence(Scalar key) {
     return new ColumnVector(mapContains(getNativeView(), key.getScalarHandle()));
   }
 
+  /** For a column of type List<Struct<_, _>> and a passed in key column, return a boolean
+   * column for all keys in the structs, It is true if the key exists in the corresponding map for
+   * that row, false otherwise. It will never return null for a row.
```

Comment:
Humnn, this seems weird to me as Spark typically wants nulls if the input is null.

Reply:
Spark isn't directly calling this method. Even if it was, we have done output processing in other methods to massage the data to conform to how Spark wants it.